docs/modules/mongodb_watchdog.md

```diff
@@ -1 +1,15 @@
 # Logger: `mongodb_watchdog`
+
```

Comment:
Start by explaining what the module does. "watchdog" is familiar to users of earlier Drupal versions, but since then people are more famililar with the "logger" name than with "watchdog", which tends to mean something else.